Source/Charts/Charts/PieRadarChartViewBase.swift

```diff
@@ -686,21 +686,21 @@ open class PieRadarChartViewBase: ChartViewBase
             {
                 break
             }
-            
```

Comment:
seeing multiple diffs here. what's the standard for us? empty line have space or not?

Reply:
I'm sorry, I don't see what you're referring to. Do you mean removing the white space on blank lines?

Reply:
I mean sometimes the empty line has space while sometime it doesn't. which is the correct way for empty line?

Reply:
An empty line should have no white space

Reply:
alright. then let's us this.